app/models/concerns/warehouse_reports/export.rb

```diff
@@ -30,7 +30,7 @@ def value_for_display(key, value)
       when 'data_source_ids'
         GrdaWarehouse::DataSource.where(id: value).map(&:short_name)
       when 'project_ids'
-        GrdaWarehouse::Hud::Project.where(id: value).map(&:name_and_type)
+        GrdaWarehouse::Hud::Project.where(id: value).map { |p| p.name(include_project_type: true, ignore_confidential_status: true) }
```

Comment:
Erring on the side of "leaving things as is" here.  This concern is currently used by `GrdaWarehouse::WarehouseReports::Exports::AdHoc` and `GrdaWarehouse::WarehouseReports::Youth::Export`